tests/tests/test-crowdloan.ts

```diff
@@ -48,23 +49,28 @@ describeDevMoonbeam("Crowdloan", (context) => {
       GENESIS_ACCOUNT
     );
     expect(isPayable.toHuman()).to.equal(null);
-    // check vesting period
-    expect((await context.polkadotApi.consts.crowdloanRewards.vestingPeriod).toHuman()).to.eq(
-      "201,600"
-    );
   });
   it("should be able to register the genesis account for reward", async function () {
     // should be able to register the genesis account for reward
     await context.polkadotApi.tx.sudo
       .sudo(
-        context.polkadotApi.tx.crowdloanRewards.initializeRewardVec(
-          [[relayChainAddress, GENESIS_ACCOUNT, 3_000_000n * GLMR]],
-          0,
-          1
-        )
+        context.polkadotApi.tx.crowdloanRewards.initializeRewardVec([
+          [relayChainAddress, GENESIS_ACCOUNT, 3_000_000n * GLMR],
+        ])
```

Comment:
so we're not doing batch tx anymore?

Reply:
This specific test never had batch, there is a bit below in the file that tests the batch calls